book_source/03_topical_pages/94_docker/05_building_images.Rmd

```diff
@@ -103,7 +103,7 @@ NOTE: All commands assume you are working from the PEcAn source code root direct
    
 3. To update to the latest local code, run `./scripts/docker_rebuild.sh`.
    Under the hood, this uses `curl` to post a RabbitMQ message to a running Docker instance.
-   By default, the scripts assumes that username and password are both `guest` and that the RabbitMQ URL is `http://localhost:8000/rabbitmq`.
+   By default, the scripts assumes that username and password are both `guest` and that the RabbitMQ URL is `http://pecan.localhost/rabbitmq`.
```

Comment:
```suggestion
   By default, the scripts assumes that username and password are both `guest` and that the RabbitMQ URL is `http://rabbitmq.pecan.localhost/`.
```

probably need to update the shell script in that case.

Reply:
Yeah, there's a separate issue for finishing this switch in code #2955